test/e2e/common/networking.go

```diff
@@ -35,10 +35,9 @@ var _ = ginkgo.Describe("[sig-network] Networking", func() {
 			Testname: Networking, intra pod http
```

Comment:
I think I am OK with this change now. One lingering question. Changing to not use hostNetwork is a substantial change to these tests. Should we be updating the Release: meta data here to include 1.16, because we are making a substantial change to the test? The reality is that the previous versions of the tests were not really even testing what they said they were testing. But I still think that we probably should list both - the docs say:
> Release: indicates the Kubernetes release that the test was added to the conformance test suite. If the test was modified in subsequent releases then those releases should be included as well (comma separated)

@timothysc @spiffxp do you agree we should update the Release to "v1.9, v1.16" here?